packages/services/api/src/modules/schema/providers/schema-publisher.ts

```diff
@@ -1674,12 +1686,9 @@ export class SchemaPublisher {
       project,
       organization,
     );
-    const comparedSchemaVersion = compareToPreviousComposableVersion
-      ? latestComposableSchemaVersion
-      : latestSchemaVersion;
-    const schemaVersionContracts = comparedSchemaVersion
+    const schemaVersionContracts = latestSchemaVersion
       ? await this.contracts.getContractVersionsForSchemaVersion({
-          schemaVersionId: comparedSchemaVersion.id,
+          schemaVersionId: latestSchemaVersion.id,
         })
       : null;
```

Comment:
same as https://github.com/kamilkisiela/graphql-hive/pull/5496#discussion_r1728545765

Reply:
got it